server.js

```diff
@@ -18,8 +19,66 @@ import env from "./config/env.generated.json";
 const defaultPort = 3000;
 const PORT = env.PORT || process.env.PORT || defaultPort;
 
+/**
+ * Some app security settings
+ */
+
+var securityHeaders = {
+  directives: {
+    defaultSrc: [
+      `\'self\'`
+    ],
+    scriptSrc: [
+      `\'self\'`,
+      `\'unsafe-inline\'`
+    ],
+    fontSrc: [
+      `\'self\'`,
+      `https://code.cdn.mozilla.net`
+    ],
+    styleSrc: [
+      `\'self\'`,
+      `\'unsafe-inline\'`,
+      `https://code.cdn.mozilla.net`
+    ],
+    imgSrc: [
+      `\'self\'`,
+      `\'unsafe-inline\'`,
+      `data:`,
```

Comment:
The reason why I put `data:` here is that the footer styling we are going to pull from `mofo-bootstrap` encodes SVG in stylesheet. That's why I *thought* we should allow `data:` here?